### PR TITLE
Fixes #25360 - two test fixups

### PR DESCRIPTION
**Pull request by beav**

### CODE REVIEW

**Comment by beav** [comment]
@jlsherrill thx!

**Comment by ekohl** [comment]
Do you want to assert there are no other entries as well?

File: test/models/activation_key_test.rb [hunk 1]
```diff
@@ -178,7 +178,8 @@ def test_available_subscriptions
       @dev_key.stubs(:get_pools).returns(cp_pools)
       @dev_key.pools = []
 
-      assert_equal [pool_one, pool_two], @dev_key.available_subscriptions
+      assert @dev_key.available_subscriptions.include? pool_one
+      assert @dev_key.available_subscriptions.include? pool_two
```
Comment (jturel) [inline test/models/activation_key_test.rb]: This could also be written using assert_includes but I don't feel strongly about it :)

**Comment by beav** [comment]
> @johnsonm325 once this is merged kick off your tests again and see how it goes

what fail are yall having? we are working through a couple

**Comment by jturel** [comment]
> what fail are yall having? we are working through a couple

that's just the one :)

**Comment by beav** [comment]
[test katello]

**Comment by beav** [comment]
@ekohl update to check array length as well

We also removed the debvcr test to confirm the error does not "move" to another test when that one isn't there

**Comment by jlsherrill** [comment]
ACK

**Pull request merged**